src/wallet/index.ts

```diff
@@ -31,6 +31,13 @@ function hexFromBuffer(buffer: Buffer): string {
   return buffer.toString('hex').toUpperCase()
 }
 
+interface WalletOptions {
+  // The classicAddress corresponding to its Regular Key Pair.
+  classicAddress?: string
+  // The seed used to derive the account keys.
+  seed?: string
+}
```

Comment:
If this is only used in the normal wallet constructor, I'm not sure what the seed option would be used for (Since the constructor already has you set up your public and private keys, which should correspond to the Regular Key that you want to sign with)

Unless I'm missing something I think we probably can just include `classicAddress` as the only optional parameter to the Wallet constructor

Reply:
`seed` has a separate use case from Regular Key. We want to store the generated `seed` after calling `Wallet.generate()`. Otherwise, the client wouldn't know what seed was used to generate the wallet.

Reply:
In this case would that be the seed for the keypair that corresponds with the `classicAddress`? (Or would the `seed` correspond to the `publicKey` and `privateKey` provided in the constructor?

(The reason I'm asking is because if `seed` corresponds to the `classicAddress` that might defeat the point of the regular keypair, which is supposed to let you do everything without the master keypair)